tools/print_test_stats.py

```diff
@@ -761,9 +762,9 @@ def assemble_s3_object(
                         'cases': {
                             name: {
                                 'seconds': case.time,
-                                'status': 'skipped' if case.skipped else
-                                          'errored' if case.errored else
-                                          'failed' if case.failed else None
+                                'status': 'errored' if case.errored else
+                                          'failed' if case.failed else 
+                                          'skipped' if case.skipped else None
```

Comment:
why rearrange these?

Reply:
This is necessarily in case error/fail occurs on the second test but skipped on first . We want to report the exception first